.env.example

```
@@ -1,41 +1,48 @@
+RUST_LOG=REPLACE_WITH_RUST_LOG
+DOWNLOAD_DIR=REPLACE_WITH_DOWNLOAD_DIR
+YTDLP_PATH=REPLACE_WITH_YTDLP_PATH
+YTDLP_EXTERNAL_DOWNLOADER=REPLACE_WITH_YTDLP_EXTERNAL_DOWNLOADER
+YTDLP_EXTERNAL_DOWNLOADER_ARGS=REPLACE_WITH_YTDLP_EXTERNAL_DOWNLOADER_ARGS
+CAPTCHA_SECRET_KEY=REPLACE_WITH_CAPTCHA_SECRET_KEY
+MAX_CONCURRENT_DOWNLOADS=REPLACE_WITH_MAX_CONCURRENT_DOWNLOADS
+MASTER_API_KEY=REPLACE_WITH_MASTER_API_KEY
+GITHUB_PAT=REPLACE_WITH_GITHUB_PAT
+GITHUB_USERNAME=REPLACE_WITH_GITHUB_USERNAME
+GITHUB_GRAPHQL_URL=https://api.github.com/graphql
+WARP_LICENSE_KEY=REPLACE_WITH_WARP_LICENSE_KEY
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WARP_LICENSE_KEY key should go before the YTDLP_EXTERNAL_DOWNLOADER key

(UnorderedKey)